breathe/renderer/sphinxrenderer.py

```diff
@@ -1847,6 +1847,15 @@ def visit_variable(self, node) -> List[Node]:
             elements.append(typename)
             elements.append(name)
             elements.append(node.get_argsstring())
+            if dom == 'cs':
+                typename = typename.replace(' ', '')
```

Comment:
@xuhongxu96 Actually, I notice `typename` here is used above on line 1847, so this change here has no effect.

I suppose we need another `if dom == 'cs'` before the `elements.append(typename)` line. Can you check?

Reply:
Yes, you're right! Sorry for the mistake. I'll fix it.